native~/Runtime/src/UnityPrepareRendererResources.cpp

```diff
@@ -1191,6 +1191,8 @@ void* UnityPrepareRendererResources::prepareInMainThread(
               if (texCoordIndexIt != primitiveInfo.uvIndexMap.end()) {
                 UnityEngine::Texture texture =
                     TextureLoader::loadTexture(gltf, baseColorTexture->index);
+                texture.hideFlags(
+                    DotNet::UnityEngine::HideFlags::HideAndDontSave);
```

Comment:
This needs to be inside the `if` below, because it's possible for `loadTexture` to return a null reference.